lms/static/sass/views/_login-register.scss

```diff
@@ -509,16 +509,16 @@
         }
 
         &.button-oa2-azuread-oauth2 {
-            color: darken($microsoft-blue, 20%);
+            color: darken($microsoft-black, 20%);
```

Comment:
@hamzamunir7300 Do you understand the problem here correctly?
It's not necessary to call `darken` on a black color already.
We don't need the `darken` call here anymore if we are changing their color to black background